includes/class-boldgrid-inspirations-deploy.php

```diff
@@ -1296,6 +1296,18 @@ public function deploy_page_sets() {
 
 		$deploy_menus = new \Boldgrid\Inspirations\Deploy\Menus( $pages_in_pageset );
 
+		/*
+		 * If the page has custom page headers, after the pages are created, we need to update the page headers.
+		 * This value starts out as false, and is set to true if we find custom page headers.
+		 */
+		$theme_has_cph = false;
+
+		/*
+		 * This variable will store the original post IDs so that they can be changed in the theme mods
+		 * at a later point.
+		 */
+		$cph_original_ids = array();
```

Comment:
Can we add just a bit more to the docblock... Maybe a quick explanation of what the key is and what the value is?

e.g. [ Author's ID ] => local id